src/index.ts

```diff
@@ -65,6 +69,7 @@ const awsNoVerifySsl = !options.verifySsl;
 const enableChromeSeamlessSso = !!options.enableChromeSeamlessSso;
 const forceRefresh = !!options.forceRefresh;
 const noDisableExtensions = !options.disableExtensions;
+const disableGpu = options.disableGpu;
```

Comment:
Let's ensure the value is strictly either `true` or `false`(See [Reference](https://github.com/aws-azure-login/aws-azure-login/blob/ead5c5ccf18bfcb3da5b521fb7e7c389c2196639/src/index.ts#L69-L70)) 
```suggestion
const disableGpu = !!options.disableGpu;
```

Reply:
Fixed that.